website/blog/2024-01-16-running-tabby-locally-with-rocm/index.md

```diff
@@ -0,0 +1,36 @@
+---
+slug: running-tabby-locally-with-rocm.md
+title: Running Tabby Locally with AMD ROCm
+authors: [boxbeam]
+tags: [deployment]
+---
+
+For those using (compatible) AMD graphics cards, you can now run Tabby locally with GPU acceleration using AMD's ROCm toolkit.
+
+Currently, Tabby with ROCm is only supported on Linux, and can only be run directly from a compiled binary. In the future, Tabby will be able to run with ROCm on Windows, and we will distribute a Docker container capable of running with ROCm on any platform.
+
```

Comment:
Consider making the following content into a new section with title  ##Tutorial

Reply:
How about making the exact steps into numbered lists for better clarity

Reply:
I converted it into a numbered list, but I don't think it looks better this way. You can let me know if you find it any easier to follow - I personally don't.

Reply:
You can also consider the structure like https://tabby.tabbyml.com/blog/deploy-tabby-to-huggingface-space.md

Reply:
I think that would make more sense